model/src/main/java/org/openremote/model/notification/Notification.java

```diff
@@ -54,6 +57,12 @@ public Target(@JsonProperty("type") TargetType type, @JsonProperty("id") String
             this.id = id;
         }
 
+        public Target(TargetType type, String id, String locale) {
```

Comment:
Can you add some JavaDocs to this class so it is clear that the `locale` contains ISO 639-1 values?